domains/hazardpro.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Ardimua7",
+        "email": "amstoregen2@gmail.com"
+    },
+    "record": {
+        "MX": ["tempm.com"]
```

Comment:
i dont think a temp email service offers MX for custom domains, please use another MX record